geocode/ngeso.py

```diff
@@ -51,6 +51,7 @@ def force_setup(self):
         Function to setup lookup files.
         """
         self._load_gsp_boundaries_20220314()
```

Comment:
```suggestion
        self.load_gsp_boundaries(version="20220314")
```

The fewer hard-coded references the better for future updates